Project.toml

```diff
@@ -14,8 +14,9 @@ julia = "1"
 
 [extras]
 Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
```

Comment:
Shouldn't you just add `SmartAsserts` here and drop `Pkg`?  Then you can just use it in the tests.